src/Panel/Ui/FilePreviews/FileDefaultPreview.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Kirby\Panel\Ui\FilePreviews;
+
+use Kirby\Cms\File;
+use Kirby\Panel\Ui\FilePreview;
+
+/**
+ * Fallback file preview component
+ *
+ * @package   Kirby Panel
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     5.0.0
+ * @internal
+ */
+class FileDefaultPreview extends FilePreview
+{
+	public string $component = 'k-file-default-preview';
+
+	/**
+	 * Accepts any file as last resort
+	 */
+	public static function accepts(File $file): bool
+	{
+		return true;
+	}
+
+	/**
+	 * Icon or image to display as thumbnail
+	 */
+	public function thumb(): array|null
+	{
+		return $this->file->panel()->image([
+			'back'  => 'transparent',
+			'ratio' => '1/1'
+		], 'cards');
+	}
```

Comment:
`Question:` What do you think about the name? So far, we have called it `image`. But I was wondering if we can find a way to differentiate between the image shown on items in overviews and this preview image shown on the file view. Or maybe there is no difference and both should be called `image` here.